docker/policies/yum_update_disabled.rego

```diff
@@ -0,0 +1,36 @@
+package appshield.DS008
+
+__rego_metadata__ := {
+	"id": "DS008",
+	"title": "Yum Update Enabled",
+	"version": "v1.0.0",
+	"severity": "CRITICAL",
+	"type": "Dockerfile Security Check",
+	"description": "Yum update is being used",
+	"recommended_actions": "Don't install unnecessary packages or run 'updates' (yum update) that downloads many files to a new image layer.",
+	"url": "https://docs.docker.com/engine/install/centos/#upgrade-docker-engine-1",
```

Comment:
This is a document regarding how to install/upgrad Docker Engine. I don't think this rule is also applied to Dockerfile.

Reply:
here is original
https://github.com/Checkmarx/kics/blob/master/assets/queries/dockerfile/yum_update_disabled/metadata.json
may be I'm missing something

Reply:
I also found mention of `yum update`
https://developers.redhat.com/blog/2016/02/24/10-things-to-avoid-in-docker-containers

Reply:
In flavor of [this blog](https://github.com/aquasecurity/appshield/pull/63#discussion_r659171869), I want to drop this policy.